maquetado/index.html

```diff
@@ -9,12 +9,15 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <div class='book'>
-    <div class='book-title'><h1 class='title'>Título del Libro</h1> <p class='title-genre'>(género)</p></div>
-    <div class='book-details'>
-      <p class='book-item'><strong>Autor del Libro: </strong> <span class='book-item-span'>del autor del Libro</span></p>
-      <p class='book-item'><strong>Editorial: </strong><span class='book-item-span'>Nombre de editorial</span></p>
-      <p class='book-item'><strong>Año de Publicación: </strong><span class='book-item-span'>Año de Publicación</span></p>
+  <div class='book-card'>
+    <img class='book-image' src='assets/book-cover.png'>
+    <div class='book'>
+      <div class='book-title'><h1 class='title'>Título del Libro</h1> <p class='title-genre'>(género)</p></div>
```

Comment:
Put in the next line the `h1` tag and the same with the tag `p`

Reply:
ok!